modules/swagger-codegen/src/main/resources/rust/model.mustache

```diff
@@ -0,0 +1,57 @@
+{{>partial_header}}
+{{#models}}
+{{#model}}
+{{#description}}
+/// {{{classname}}} : {{{description}}}
+{{/description}}
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct {{classname}} {
+{{#vars}}
+  {{#description}}
+  /// {{{description}}}
+  {{/description}}
+  #[serde(rename = "{{baseName}}")] {{name}}: {{^required}}Option<{{/required}}{{^isPrimitiveType}}{{^isContainer}}super::{{/isContainer}}{{/isPrimitiveType}}{{{datatype}}}{{^required}}>{{/required}}{{#hasMore}},{{/hasMore}}
```

Comment:
in here, if it's an array, it's rendered as e.g. `Option<Vec<Tag>>`, `super::` is missing from the child struct name.